dashboard/test/controllers/followers_controller_test.rb

```diff
@@ -253,12 +253,10 @@ class FollowersControllerTest < ActionController::TestCase
   test "create when already followed by a teacher switches sections" do
     sign_in @laurel_student_1.student_user
 
-    assert_does_not_create(Follower) do
-      post :create, params: {
-        section_code: @laurel_section_2.code,
-        redirect: '/'
-      }
-    end
```

Comment:
The test should create and destroy a `Follower`, so it is misleading to assert that it does not create one.